index.html

```diff
@@ -0,0 +1,350 @@
+              <li class="pricing-stats">25 Calendar</li>
+            </ul>
+            <div class="pricing-button-container">
+              <a href=""><button class="pricing-button">SIGN UP</button></a>
+            </div>
+          </section>
+          <section class="pricing-tables">
+            <h4 class="pricing-header white-color">PREMIUM</h4>
+            <ul>
+              <li class="pricing-plan"><span class="pricing-cost">$49.99</span> /MONTH</li>
+              <li class="pricing-stats">UNLIMITED Projects</li>
+              <li class="pricing-stats">UNLIMITED Users</li>
+              <li class="pricing-stats">1000 Gb User</li>
+              <li class="pricing-stats">UNLIMITED Calendar</li>
+            </ul>
+            <div class="pricing-button-container">
+              <a href=""><button class="pricing-button">SIGN UP</button></a>
+            </div>
+          </section>
+        </section>
+      </article>
+      <article id="portfolio" class="portfolio-container section-arrows-orange">
+        <h2 class="portfolio-section-header gray-line">PORTFOLIO</h2>
+        <div class="portfolio-slider">
+          <figure class="portfolio-slider-figure">
+            <img class="portfolio-slider-overlay" src="/img/slider.jpg" width="940" height="336" alt="image advertising Crispers iphone app" title="Crispers/Clouders app">
+            <figcaption class="portfolio-slider-caption">
+              <h3 class="white-color">Crispers</h3>
+              <p>Nice, crispy and creative landing page</p>
+              <a href="http://google.com"><button class="portfolio-slider-button">GET THE APP</button></a>
+              <p>Available for iPhone and iPad</p>
+            </figcaption>
+          </figure>
+          <ul class="portfolio-slider-bullets" title="slider bullets" aria-hidden="true">
+            <li title="first bullet"></li>
+            <li title="second bullet"></li>
+            <li title="third bullet"></li>
+            <li title="fourth bullet" class="slider-bullet-active"></li>
+            <li title="fifth bullet"></li>
+          </ul>
+        </div>
+
+        <nav title="portfolio navigation">
+          <ul title="portfolio-categories">
+            <li><a class="portfolio-menu-item portfolio-selected-category" href="">All works</a></li>
+            <li><a class="portfolio-menu-item" href="">Webdesign</a></li>
+            <li><a class="portfolio-menu-item" href="">UI design</a></li>
+            <li><a class="portfolio-menu-item" href="">Icon design</a></li>
+            <li><a class="portfolio-menu-item" href="">Brand &amp; Logotypes</a></li>
+          </ul>
+        </nav>
+
+        <figure class="portfolio-item">
+          <a class="portfolio-link-hover" href=""><img class="portfolio-opacity" src="/img/ytube-app.jpg" alt="Image of Youtube app on a handheld device" title="Youtube app image"></a>
+          <figcaption>
+            <h4 class="portfolio-caption-header">Youtube app</h4>
```

Comment:
Make sure your copy matches that of the comp's -- that's something that any product owner will pick up on, and if the copy's wrong, then it makes them wonder what else might have been missed.